drivers/wdm/audio/backpln/portcls/dma_slave.cpp

```diff
@@ -213,36 +194,20 @@ CDmaChannelInit::MaximumBufferSize()
     return m_MaximumBufferSize;
 }
 
-#ifdef _MSC_VER
-
-PHYSICAL_ADDRESS
-NTAPI
-CDmaChannelInit::PhysicalAddress()
-{
-    DPRINT("CDmaChannelInit_PhysicalAddress: this %p Virtual %p Physical High %x Low %x%\n", this, m_Buffer, m_Address.HighPart, m_Address.LowPart);
-
-    return m_Address;
-}
-
-#else
-
 PHYSICAL_ADDRESS
 NTAPI
 CDmaChannelInit::PhysicalAddress(
     PPHYSICAL_ADDRESS Address)
 {
-    DPRINT("CDmaChannelInit_PhysicalAddress: this %p Virtual %p Physical High %x Low %x%\n", this, m_Buffer, m_Address.HighPart, m_Address.LowPart);
+    DPRINT("CDmaChannelInit_PhysicalAdress: this %p Virtuell %p Physical High %x Low %x%\n", this, m_Buffer, m_Address.HighPart, m_Address.LowPart);
```

Comment:
Strip that change. It is contraproductive / wrong.

Reply:
Fixed.